ptx/sec_limit_continuity.ptx

```diff
@@ -956,14 +966,16 @@
               \end{tikzpicture}
             </latex-image>
             <description>
-              Graph showing the domain 0 to 4 and range 0 to 100. There is a 
-              vertical dotted line at <m>x = 2</m> illustrating an asymptote.
-              As <m>x</m> approaches 2 from both sides the value of <m>f(x)</m>
-              approaches infinity. The graph also asymptotes at <m>y = 0</m>.
-              On both sides of the dotted vertical line the graph has an upward 
-              curve with an increasing slope at <m>x</m> gets closer to 2.
-              The asymptote at <m>x = 2</m> causes there to be an infinite
-              discontinuity.
+              <p>
+                Graph showing the domain 0 to 4 and range 0 to 100. There is a 
+                vertical dotted line at <m>x = 2</m> illustrating an asymptote.
+                As <m>x</m> approaches 2 from both sides the value of <m>f(x)</m>
+                approaches infinity. The graph also asymptotes at <m>y = 0</m>.
```

Comment:
I don't think you can use 'asymptote' as a verb. Do you maybe mean "approaches the asymptote"?